pkgs/development/python-modules/mkdocs-markdownextradata-plugin/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, mkdocs
+, pytestCheckHook
+, pythonOlder
+, pyyaml
+, click
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-markdownextradata-plugin";
+  version = "0.2.4";
```

Comment:
This tools seems abandoned as it hasn't received any updates for 2 years. Are you sure we should include this in Nixpkgs?

Reply:
Your right! 

Going to use the packaged [`mkdocs-macros`](https://github.com/fralau/mkdocs_macros_plugin) plugin.

Closing this PR now.